yabt/utils.py

```diff
@@ -150,7 +148,8 @@ def link_files(files: set, workspace_src_dir: str,
         abs_src = join(conf.project_root, src)
         abs_dest = join(conf.project_root, workspace_src_dir,
                         relpath(src, base_dir))
-        link_node(abs_src, abs_dest, True)
+        force = 'yabtwork' in src
+        link_node(abs_src, abs_dest, force)
```

Comment:
what's the value of `force` as temp var? 

for readability you could do `force=conf.builders_workspace_dir in src`